tests/Calculator/CalculatorTest.php

```diff
@@ -84,6 +84,10 @@ public function testRound()
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_EVEN));
         $this->assertEquals('2', $calculator->round(2, Money::ROUND_HALF_ODD));
         $this->assertEquals('12', $calculator->round('12.50', Money::ROUND_HALF_DOWN));
+        $this->assertEquals('-2', $calculator->round(-1.5, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.578947368, Money::ROUND_HALF_UP));
+        $this->assertEquals('-8329', $calculator->round(-8328.5, Money::ROUND_HALF_UP));
```

Comment:
In fact, rounding half up should result in `-8328.5` moved towards positive infinity which results in `-8328`. That's the way PHP behaves too, but with introducing our custom constraints, we can fix that issue and provide correct implementations (I did something like that some time ago in [Rounder](https://github.com/pamil/Rounder)).